### PR TITLE
chore(RELEASE-1085): fix shellcheck - rh-sign-image

### PR DESCRIPTION
**Pull request by scoheb**
## Describe your changes

- fix linting issues in rh-sign-image

## Relevant Jira

- [RELEASE-1085](https://issues.redhat.com//browse/RELEASE-1085)

## Checklist before requesting a review
- [x] I have marked as draft or added `do not merge` label if there's a dependency PR
  - If you want reviews on your draft PR, you can add reviewers or add the `release-service-maintainers` handle if you are unsure who to tag
- [x] My commit message includes `Signed-off-by: My name <email>`
- [x] I have bumped the task/pipeline version string and updated changelog in the relevant README
- [x] I read CONTRIBUTING.MD and [commit formatting](CONTRIBUTING.md#commit-message-formatting-and-standards)

### CODE REVIEW

File: tasks/rh-sign-image/rh-sign-image.yaml [hunk 1]
```diff
@@ -185,8 +184,8 @@ spec:
             # Push source container if the component has pushSourceContainer: true or if the
             # pushSourceContainer key is missing from the component and the defaults has
             # pushSourceContainer: true
-            if [[ $(jq -r ".components[$COMPONENTS_INDEX].pushSourceContainer" ${SNAPSHOT_PATH}) == "true" ]] || \
-               [[ $(jq ".components[$COMPONENTS_INDEX] | has(\"pushSourceContainer\")" ${SNAPSHOT_PATH}) == "false" && \
+            if [[ $(jq -r ".components[$COMPONENTS_INDEX].pushSourceContainer" "${SNAPSHOT_PATH}") == "true" ]] || \
+               [[ $(jq ".components[$COMPONENTS_INDEX] | has(\"pushSourceContainer\")" "${SNAPSHOT_PATH}") == "false" && \
```
Comment (johnbieren) [inline tasks/rh-sign-image/rh-sign-image.yaml]: Fails yamllint on this line now

File: .github/workflows/lint.yaml
Comment (johnbieren) [inline .github/workflows/lint.yaml:1]: You can remove the changes to this file once PR is ready

**Review by johnbieren — COMMENTED** [review]
Need to fix the commit title. Can't have capital F in Fix and missing :

File: .github/workflows/lint.yaml
Comment (scoheb) [reply .github/workflows/lint.yaml:1]: fixed

**Review by johnbieren — APPROVED** [review]
thanks for picking this up

**Review by mmalina — APPROVED** [review]

**Comment by mmalina** [comment]
FYI, stage Pyxis is down ATM, so e2e is failing.

**Comment by mmalina** [comment]
Stage pyxis is back up now.

**Pull request merged**